app/background.js

```diff
@@ -3,7 +3,7 @@ const runtime = chrome.runtime;
 const tabs = chrome.tabs;
 const browserAction = chrome.browserAction;
 
-const githubPattern = /^https:\/\/([^\.]+\.)*github\.com/;
+const githubPattern = /^https:\/\/([^\.]+\.)*github*/;
```

Comment:
they can't for people that run github internally, and use domains like https://github.foocompany.com